.github/workflows/acc-dockeronly-nightly.yml

```diff
@@ -0,0 +1,30 @@
+name: Acceptance Tests Docker Only
```

Comment:
We've been trying to standardize a bit on names in other pull requests that are touching this directory lately. The only rule for this would be to prefix it with `test`, though we might be able to future proof the name based on the context provided in the PR message.

Perhaps something like `test-go-acceptance-nightly`. I'm not sure whether or not we only run docker at this point is relevant, as that is bound to change shortly.

As per the callable workflow. We might try and do the same thing, eg:
`test-run-go-tests-for-path`.